#! /bin/sh /usr/share/dpatch/dpatch-run
## 11-promote-QtResource-to-qt4.dpatch by Marek Brudka <mbrudka@aster.pl>
##
## DP: Moves QtResources dependencies from qt3 to qt4 [should be added to the upstream]

@DPATCH@
--- ACE_wrappers/ace/QtReactor/ace_qt3reactor.mpc	2010-09-17 22:16:53.000000000 +0200
+++ /dev/null	2010-12-30 21:38:16.809455563 +0100
@@ -1,27 +0,0 @@
-//  -*- MPC -*- now wouldn't this be cool...
-// $Id$
-
-project(ACE_Qt3Reactor_moc) : acedefaults, ace_qt3 {
-  custom_only = 1
-  MOC_Files {
-    QtReactor.h
-  }
-}
-
-project(ACE_Qt3Reactor) : acedefaults, install, acelib, ace_output, ace_qt3 {
-  after       += ACE_Qt3Reactor_moc
-  sharedname   = ACE_QtReactor
-  dynamicflags += ACE_QTREACTOR_BUILD_DLL
-
-  MOC_Files {
-  }
-
-  Source_Files {
-    QtReactor_moc.cpp
-    QtReactor.cpp
-  }
-
-  specific {
-    install_dir = ace/QtReactor
-  }
-}
